lib/media/time_ranges_utils.js

```diff
@@ -23,7 +23,8 @@ shaka.media.TimeRangesUtils = class {
       return null;
     }
     // Workaround Safari bug: https://bit.ly/2trx6O8
-    if (b.length == 1 && b.end(0) - b.start(0) < 1e-6) {
+    // Firefox may leave <1e-4s of data in buffer after clearing all content
+    if (b.length == 1 && b.end(0) - b.start(0) < 1e-4) {
```

Comment:
There are lots of those checks... Maybe move it to some util function so in the future we will not be forced to change the same value all over the place?

Reply:
Yes, I agree, please move the check to a helper function. Thanks!